nextflow.config

```diff
@@ -1,6 +1,15 @@
 process {
     executor = 'slurm'
     cache = 'lenient'
+    resourceLimits {
+        // These limits are the maximum across all partitions and QOS
+        // on the cluster. Thus, these are mostly taken from the bonus QOS, which
+        // has the most lenient resource limits.
+        // See `sacctmgr show QOS Format=name%15,MaxTRESPU%110,MaxWall`
+        cpus = 6896
+        memory = 45548110.M
```

Comment:
M needs to be MB. M was being interepreted as minutes and I was getting
`Cannot cast object '31630d 15h 10m' with class 'nextflow.util.Duration' to class 'nextflow.util.MemoryUnit'`
(45548110/60/24 = 31640.something days)